### PR TITLE
AWS Service Control Policy - Pagination Fix

### PR DESCRIPTION
**Pull request by srpomeroy**
### Description

Fixes pagination to add the `NextToken` to the body instead of query params

### Issues Resolved

N/A

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: compliance/aws/scp_audit/aws_scp_audit.pt [hunk 1]
```diff
@@ -48,7 +48,7 @@ pagination "pagination_aws_json" do
     body_path jmes_path(response, "NextToken")
   end
   set_page_marker do
-    query "NextToken"
+    body_field "NextToken"
```
Comment (douglaswth) [inline compliance/aws/scp_audit/aws_scp_audit.pt:51]: Looks good! I just noticed below that there is the following in the `ds_org_accounts` datasource:

```ruby
body_field "empty", "empy"
```

That's a little hacky, so while you are here, I think you could change it to an empty JSON object with:

```ruby
body "{}"
```

Comment (srpomeroy) [reply compliance/aws/scp_audit/aws_scp_audit.pt:51]: You learn something new every day. Thanks @douglaswth!

Comment (douglaswth) [reply compliance/aws/scp_audit/aws_scp_audit.pt:51]: 🤣 I actually implemented it two years ago so I was surprised to see that there were workarounds for the lack of it, but then I checked the docs and apparently it never made it in there, so now I'm going to add it there as well.

**Review by douglaswth — APPROVED** [review]
LGTM

**Pull request merged**